src/language.js

```diff
@@ -10,8 +10,9 @@ const getLanguageCodeFromEnglishName = language => {
 };
 
 const findLanguageCode = language => {
-  let languageByCode = (iso_639_2[language] || {})["639-2"];
-  return languageByCode || getLanguageCodeFromEnglishName(language) || null;
+  let languageByThreeLetterCode = (iso_639_2[language] || {})["639-2"];
+  let languageByTwoLetterCode = (iso_639_1[language] || {})["639-2"];
```

Comment:
let languageByTwoLetterCode = (iso_639_1[language] || {})["639-1"];

Reply:
Why? it should still return the three letter language code.

If you look one line bellow, this function allows to get three letter code (ISO-639-2) in 3 ways:
- by providing three letter code (it validates if it exists)
- by providing two letter code
- by providing english name of the language

Reply:
That's right.

Reply:
Isn't this going to be backwards incompatible for users of our package? Especially that this be more of a ticking time bomb, as it will not immediately break, but at some random point in time when the service starts returning different values.

Reply:
No, this change is not backwards incompatible as it only provides an additional way to query for results. The client still passes the same request to the service.

However, based on other comments I see that adding this option is not so helpful, because it is not changing what the client returns.

Probably the way to go will be adding a new method, which clearly indicates what is going to happen. Pass 2 letter code -> get 2 letter code back from the service.

I will consider whether it is better to support that directly by the service, or if the client can handle this transformation.